src/main/java/staffconnect/model/availability/Availability.java

```diff
@@ -4,18 +4,36 @@
 import static staffconnect.commons.util.AppUtil.checkArgument;
 
 import java.time.DayOfWeek;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 /**
  * Represents a Person's availability in the staff book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAvailability(String)}
  */
 public class Availability {
 
 
-    public static final String MESSAGE_CONSTRAINTS =
-            "Should be a week of the day, the full word or the first syllable of the word";
-    public static final String VALIDATION_REGEX = "(?i)((mon|tue(s)?|wed(nes)?|thu(r)?(rs)?|fri|sat(ur)?|sun)(day)?)";
+    public static final String MESSAGE_CONSTRAINTS = "[DAY_OF_THE_WEEK] [START_TIME] [END_TIME] E.g. mon 09:00 13:00\n"
+            + "[DAY_OF_THE_WEEK] Should be a week of the day, the full word or the first syllable of the word\n"
+            + "[START_TIME], [END_TIME] Using a 24H digital time format.\n";
+    public static final String MESSAGE_INVALID_TIME = "Invalid Time, Format: HH:mm\n"
+            + "End time must be after Start time";
+    public static final String MESSAGE_END_BEFORE_START = "Format: [DAY_OF_THE_WEEK] [START_TIME] [END_TIME]\n"
+            + "End time must be after Start time";
+    public static final String VALIDATION_REGEX_DAY = "(?i)((mon|tue(s)?|wed(nes)?|thu(r)?(rs)?|fri|sat(ur)?|sun)"
+            + "(day)?)";
+    public static final String VALIDATION_REGEX_TIME = "([01][0-9]|2[0-3]):([0-5][0-9])";
+    public static final String VALIDATION_REGEX = VALIDATION_REGEX_DAY + " [0-9]{2}:[0-9]{2} [0-9]{2}:[0-9]{2}";
```

Comment:
I think you missed out on this, I don't see ```VALIDATION_REGEX_TIME``` being used anywhere

Reply:
Fixed: removed